src/stories/Button.stories.js

```diff
@@ -1,39 +0,0 @@
-import MyButton from './Button.vue'
```

Comment:
I deleted the sample Storybook stories because:
- we don't use them
- they cause runtime errors